src/common/Page.ts

```diff
@@ -1443,6 +1443,41 @@ export class Page extends EventEmitter {
     }
   }
 
+  /**
+   * Overrides the Idle state.
+   *
+   * @example
+   * ```js
+   * await page.setIdleOverride({isUserActive: true, isScreenUnlocked: false});
+   * ```
+   *
+   * @param isUserActive Mock isUserActive
+   * @param isScreenUnlocked Mock isScreenUnlocked
+   */
+  async setIdleOverride(
+    isUserActive: boolean,
+    isScreenUnlocked: boolean
+  ): Promise<void> {
+    await this._client.send('Emulation.setIdleOverride', {
+      isUserActive,
+      isScreenUnlocked,
+    });
+  }
+
+  /**
+   * Clears Idle state overrides.
+   *
+   * @example
+   * ```js
+   * await page.setIdleOverride({isUserActive: true, isScreenUnlocked: false});
+   * // do some checks here
+   * await page.clearIdleOverride();
+   * ```
+   */
+  async clearIdleOverride(): Promise<void> {
```

Comment:
For other emulation APIs we tend to not have a separate `clear` API in Puppeteer, and instead we clear when the same method is called with an empty string or `null`/`undefined` or similar, depending on the situation. In this case, we could go with `page.emulateIdleState()` (i.e. `undefined` clears all overrides). WDYT?

Reply:
done